{"id":10456,"className":"java.sql.SQLSyntaxErrorException","methodName":"SQLSyntaxErrorException","signature":"\u003cjava.sql.SQLSyntaxErrorException: void SQLSyntaxErrorException(java.lang.String,java.lang.String,int)\u003e","javadoc":"/** \n * Constructs a \u003ccode\u003eSQLSyntaxErrorException\u003c/code\u003e object with a given \u003ccode\u003ereason\u003c/code\u003e, \u003ccode\u003eSQLState\u003c/code\u003e  and \u003ccode\u003evendorCode\u003c/code\u003e. The \u003ccode\u003ecause\u003c/code\u003e is not initialized, and may subsequently be initialized by a call to the {@link Throwable#initCause(java.lang.Throwable)} method.\u003cp\u003e\n * @param reason a description of the exception\n * @param SQLState an XOPEN or SQL:2003 code identifying the exception\n * @param vendorCode a database vendor specific exception code\n * @since 1.6\n */\n","tf":{"freq":{"call":1,"reason":2,"link":1,"identifi":1,"object":1,"lang":1,"given":1,"initi":2,"subsequ":1,"except":5,"may":1,"error":2,"vendor":3,"descript":1,"init":1,"databas":1,"specif":1,"syntax":1,"caus":2,"lstate":2,"java":1,"method":1,"construct":1,"throwabl":2,"lsyntax":1},"maxFreq":5,"totalTerms":37},"loc":3}